src/pages/journal/components/slot/slot.ts

```diff
@@ -1,3 +1,4 @@
 export interface SlotComponent {
   slot: any;
+  slotChanged: boolean;
```

Comment:
`hasSlotChanged` would be a better indication that the variable holds a boolean

Reply:
Done.